### PR TITLE
Adding basic goatcounter

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

#532 

## Changes

- Adds `next-goatcounter` which is just a clean util to add goatcounter to a next app easily
- Points to our goatcounter instance

### CODE REVIEW

**Comment by devinmatte** [comment]
Also opened https://github.com/haideralipunjabi/next-goatcounter/issues/5 to be able to remove redundant `scriptSrc` line, as it shouldn't be needed

**Comment by devinmatte** [comment]
Waiting on https://github.com/haideralipunjabi/next-goatcounter/pull/6 / https://github.com/haideralipunjabi/next-goatcounter/issues/5

**Comment by devinmatte** [comment]
This is ready to go now

File: pages/_app.tsx [hunk 1]
```diff
@@ -23,6 +25,7 @@ export default function App({ Component, pageProps }) {
   return (
     <Layout>
       <DashboardLayout>
+        <GCScript siteUrl={'https://transitmatters-dd.goatcounter.com/count'} />
```
[the remark this answers is not in the document]
Comment (devinmatte) [reply pages/_app.tsx]: Good point, added a check like we do in v3

File: common/utils/constants.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-export const PRODUCTION = 'dashboard-v4.transitmatters.org';
+export const PRODUCTION = 'dashboard.transitmatters.org';
```
Comment (PatrickCleary) [inline common/utils/constants.ts:1]: Was this just an error?

**Review by PatrickCleary — APPROVED** [review]

File: common/utils/constants.ts (hunk 2 above)
Comment (devinmatte) [reply common/utils/constants.ts:1]: The prod address won't have a `v4` in the url so it was likely an error at first setup

**Pull request merged**